common/djangoapps/student/models.py

```diff
@@ -1291,6 +1311,54 @@ def is_verified_enrollment(self):
         return CourseMode.is_verified_slug(self.mode)
 
 
+class ManualEnrollmentAudit(models.Model):
+    """
+    Table for track which enrollments were performed through manual enrollment.
+    """
+    enrollment = models.ForeignKey(CourseEnrollment, null=True)
+    enrolled_by = models.ForeignKey(User, null=True)
+    enrolled_email = models.CharField(max_length=255, db_index=True)
```

Comment:
Just to be clear, we use this field for when someone is entered into the AllowedEnrollments table but he/she has not actually gone through the enrollment process. Right?